core/src/main/java/org/mskcc/cbio/portal/dao/DaoGene.java

```diff
@@ -415,6 +467,12 @@ public static void deleteAllRecords() throws DaoException {
         deleteAllAliasRecords();
     }
     
+    /**
+     * 
+     * @throws DaoException
+     * 
+     * @deprecated only used by deprecated code, so deprecating this as well.
```

Comment:
Would these comments be sufficient to tell people who are considering using the function what to do instead, and why?